### PR TITLE
Render sprites

### LINKED ISSUE #6: Draw sprites on screen
We need to draw some basic sprites (e.g. **Base** and **Builder**) according to their in-game positions.

### PR DESCRIPTION
**Pull request by ForNeVeR**
Also enables unit tests.

Close #6.

### CODE REVIEW

**Comment by ForNeVeR** [comment]
Mr. @gsomix, could you please review this?

**Comment by gsomix** [comment]
Yes, ASAP.

File: MarsBaseBuilder/Renderer.fs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    { sp with x = sp.x + x; y = sp.y + y }
+
+let baseRadius = 5<screenPx>
+let internal draw coords = function
+    | Base -> 
+        let pos = mapToScreenPoint coords
+        Rectangle (offset (-baseRadius) (-baseRadius) pos, offset baseRadius baseRadius pos)
+
+let commands (state : GameLogic.GameState) : ResizeArray<DrawCommand> =
+    let list = ResizeArray()
```
Comment (gsomix) [inline MarsBaseBuilder/Renderer.fs]: Is ResizeArray here for performance considerations?

File: MarsBaseBuilder/Measures.fs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿module MarsBaseBuilder.Measures
+
+[<Measure>]
+type gameUnit
```
Comment (gsomix) [inline MarsBaseBuilder/Measures.fs]: Interference with type GameUnit. If it's unit of position maybe better use real SI units like `km`, `m`.

File: MarsBaseBuilder/Renderer.fs (hunk 1 above)
Comment (ForNeVeR) [inline MarsBaseBuilder/Renderer.fs]: Yes, it is. Everything that get called from `Draw` method is extremely performance-sensitive and should generate as little garbage as possible. I don't intend to actually modify this `ResizeArray` outside of this function, but we need it to keep its type due to performance considerations (`System.List`'s `GetEnumerator` returns `struct` and that's important).

You may argue that it's a premature optimization, but I think that current setup should be okay for a game engine. I have a fear that we'll need to optimize it more in the future.

File: MarsBaseBuilder/GraphicsContext.fs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    interface IDisposable with
+        member __.Dispose() =
+            singlePixelTexture.Dispose()
+            spriteBatch.Dispose()
+
+    interface IGraphicsContext with
+        member __.Clear(color : Color) : unit =
+            device.Clear color
+    
+        // TODO[F]: Extract these to a separate, more safe type
```
Comment (ForNeVeR) [inline MarsBaseBuilder/GraphicsContext.fs]: Oh, I need to do this before merging.

File: MarsBaseBuilder/Measures.fs (hunk 2 above)
Comment (gsomix) [inline MarsBaseBuilder/Measures.fs]: I'm fine with `dm`.

**Pull request merged**